packages/services/examples/node/main.py

```diff
@@ -4,8 +4,8 @@
 from __future__ import print_function, absolute_import
 import json
 import os
-from jupyterlab.process import which
-from jupyterlab.process_app import ProcessApp
+from jupyterlab_launcher.process import which
+from jupyterlab_launcher.process_app import ProcessApp
```

Comment:
This is the kind of clear indication that the functionality of which and ProcessApp belong outside of jupyterlab, since otherwise this standalone example services package still has a dependency on all of jupyterlab.

Reply:
Very good point.